PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,15 @@
 <!--
 Thanks for contributing a pull request! Please ensure you have taken a look at
-the contribution guidelines: https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md#Contributing-Pull-Requests
+the contribution guidelines: https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md
+-->
+
+#### Reference Issues/PRs
+<!--
+Example: Fixes #1234. Closes #2345. See also #3456.
```

Comment:
Maybe use only only Fix or only Close (here and everywhere else). Fix and Close do the same and the user can look at the link for the list of possible keywords if he wants some variety.